.claude/skills/gemini-grade-review/review_tools.py

```
@@ -0,0 +1,126 @@
+from review_lib import (
+    dedup, dependency_findings, normalize, parse_diff, post_review,
+    sh, split_anchored, truncate_diff,
+)
+
+
+def _fetch_diff(repo, pr, base):
+    if pr:
+        return sh(["gh", "pr", "diff", str(pr), "--repo", repo])
+    if base:
+        return sh(["git", "diff", f"{base}...HEAD"])
+    sys.exit("prep 需要 --pr <num>（PR 模式）或 --base <ref>（本機分支模式）")
```

Ruff (0.15.20)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)